Documentation/Concepts/Backend/SystemModules/Index.rst

```diff
@@ -1,77 +1,113 @@
+:navigation-title: System modules
 .. include:: /Includes.rst.txt
-.. index:: Modules; System
 .. _system-modules:
 
 ==============
 System modules
 ==============
 
-.. important::
+The backend modules in group system are only available to backend users with
+admin permissions.
 
```

Comment:
```suggestion

In TYPO3 you can decide which package you want to include in your 
TYPO3 instance.
```

Reply:
This is out of context here as we are describing the modules functionality. I would describe such somewhere in the extensions chapteres